spec/requests/admin/configs_spec.rb

```diff
@@ -78,6 +78,25 @@
                                           confirmation: confirmation_message }
           expect(SiteConfig.authentication_providers).to eq([provider])
         end
+
+        context "when authentication providers are present" do
+          before do
+            SiteConfig.authentication_providers = Authentication::Providers.available.map(&:to_s)
```

Comment:
this should use `allow(SiteConfig).to receive(:authentication_providers).and_return(...)` to avoid setting it for other specs, as it's going to persist and if we change order this might have an impadct on flakey specs